pkgs/top-level/aliases.nix

```diff
@@ -1132,6 +1132,7 @@ mapAliases ({
   openjfx19 = throw "OpenJFX 19 was removed as it has reached its end of life"; # Added 2024-08-01
   openjfx20 = throw "OpenJFX 20 was removed as it has reached its end of life"; # Added 2024-08-01
   openjpeg_2 = openjpeg; # Added 2021-01-25
+  openlens = lens; # Added 2024-09-04
```

Comment:
Is it the same app / 100% compatible? I'd rather not have people surprised by the change from openlens to lens

Reply:
Throw sounds safer, yeah